api/changes/ana_fix-emoney-gasprices

```diff
@@ -0,0 +1 @@
+[Fixed] [#5182](https://github.com/cosmos/lunie/pull/5182) Fix e-Money gasprices by changing column name from gasPrice to price @Bitcoinera
```

Comment:
why was this necessary?

Reply:
it wouldn't even open up the ActionModal. Gives the following error:

![image](https://user-images.githubusercontent.com/40721795/98128730-8ccd1b80-1eb8-11eb-8a22-dd1788864c3f.png)